src/model.ts

```diff
@@ -189,9 +190,13 @@ export default class EditableDSVModel extends MutableDataModel {
     const model = this.dsvModel;
     const rowRemovedIndex = model.getOffsetIndex(rowNumber + 1, 0);
     const rowAfterIndex = model.getOffsetIndex(rowNumber + 2, 0);
-    model.rawData =
-      model.rawData.slice(0, rowRemovedIndex) +
-      model.rawData.slice(rowAfterIndex);
+    if (!rowAfterIndex) {
+      model.rawData = model.rawData.slice(0, rowRemovedIndex);
+    } else {
+      model.rawData =
+        model.rawData.slice(0, rowRemovedIndex) +
+        model.rawData.slice(rowAfterIndex);
+    }
```

Comment:
What do you think about using a ternary? 
```
model.rawData = rowAfterIndex ?
 model.rawData.slice(0, rowRemovedIndex) + model.rawData.slice(rowAfterIndex) :
 model.rawData.slice(0, rowRemovedIndex)```

If this looks too confusing, then feel free to disregard this

Reply:
Ternary looks fine to me. Cleaner. Shorter.